dotCMS/src/main/java/com/dotcms/analytics/track/PagesAndUrlMapsRequestMatcher.java

```diff
@@ -0,0 +1,48 @@
+package com.dotcms.analytics.track;
+
+import com.dotcms.visitor.filter.characteristics.Character;
+import com.dotcms.visitor.filter.characteristics.CharacterWebAPI;
+import com.dotmarketing.business.web.WebAPILocator;
+import com.dotmarketing.filters.CMSFilter;
+
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.util.Objects;
+
+/**
+ * Matcher for pages or files
+ * @author jsanca
+ */
+public class PagesAndUrlMapsRequestMatcher implements RequestMatcher {
```

Comment:
Really it is just Pages, I think we need another one for URLMap

I think we can check this attribute

https://github.com/dotCMS/core/blob/bffd903770f86a8ac2da2d7e5ef07f0e158e22dc/dotCMS/src/main/java/com/dotmarketing/portlets/htmlpageasset/business/render/HTMLPageAssetRenderedAPIImpl.java#L448

to know if the URL was processed using URLMAP

Reply:
The character already realize in advance if the page is an url map